aggregators/aggregator_test.go

```diff
@@ -709,6 +689,7 @@ func TestCombinedMetricsKeyOrderedByProjectID(t *testing.T) {
 }
 
 func TestHarvest(t *testing.T) {
+	t.Skip()
```

Comment:
Did you mean to leave this skipped?

Reply:
Ah, thanks for catching it. Not intended, fixed now.